<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>617</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ProcessBox</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../微信公众号图文/大四毕业设计/UI设计/皮肤主题素材/截图29.jpg</normaloff>../../微信公众号图文/大四毕业设计/UI设计/皮肤主题素材/截图29.jpg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="win2">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>3</number>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QSplitter" name="splitter_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QSplitter" name="splitter_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="QToolBox" name="toolBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>160</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>160</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>算法区</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::Box</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="lineWidth">
                <number>2</number>
               </property>
               <property name="currentIndex">
                <number>4</number>
               </property>
               <widget class="QWidget" name="page_3">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>152</width>
                  <height>354</height>
                 </rect>
                </property>
                <attribute name="label">
                 <string>D档part1</string>
                </attribute>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QWidget" name="widget_3" native="true">
                   <layout class="QVBoxLayout" name="verticalLayout_2">
                    <item>
                     <widget class="QPushButton" name="pushButton_5">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>60</width>
                        <height>60</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>160</width>
                        <height>60</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>灰度化</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_4">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>60</width>
                        <height>60</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>160</width>
                        <height>60</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>二值化</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_6">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>60</width>
                        <height>60</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>160</width>
                        <height>60</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>中值滤波</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="page_4">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>152</width>
                  <height>354</height>
                 </rect>
                </property>
                <attribute name="label">
                 <string>D档part2</string>
                </attribute>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QWidget" name="widget_4" native="true">
                   <layout class="QVBoxLayout" name="verticalLayout_4">
                    <item>
                     <widget class="QPushButton" name="pushButton_8">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>60</width>
                        <height>60</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>160</width>
                        <height>60</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>均值滤波</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_9">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>60</width>
                        <height>60</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>160</width>
                        <height>60</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>拉普拉斯锐化</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_10">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>60</width>
                        <height>60</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>160</width>
                        <height>60</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>直方图</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_11">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>60</width>
                        <height>60</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>160</width>
                        <height>60</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>边缘检测</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="page_5">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>152</width>
                  <height>354</height>
                 </rect>
                </property>
                <attribute name="label">
                 <string>B档part1</string>
                </attribute>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QWidget" name="widget_5" native="true">
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <item>
                     <widget class="QPushButton" name="pushButton_12">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>60</width>
                        <height>60</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>160</width>
                        <height>60</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>video_GRAY</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_13">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>60</width>
                        <height>60</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>160</width>
                        <height>60</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>video_Binay</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_14">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>60</width>
                        <height>60</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>160</width>
                        <height>60</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>video_avarage_filter</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="pushButton_15">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>60</width>
                        <height>60</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>160</width>
                        <height>60</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>video_median filter</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="page_6">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>203</width>
                  <height>333</height>
                 </rect>
                </property>
                <attribute name="label">
                 <string>B档part2</string>
                </attribute>
                <layout class="QVBoxLayout" name="verticalLayout_11">
                 <item>
                  <widget class="QWidget" name="widget_6" native="true">
                   <layout class="QVBoxLayout" name="verticalLayout_12">
                    <item>
                     <widget class="QPushButton" name="algoButton_1">
                      <property name="text">
                       <string>B档Laplace sharpening</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="page_7">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>152</width>
                  <height>354</height>
                 </rect>
                </property>
                <attribute name="label">
                 <string>A档</string>
                </attribute>
                <widget class="QPushButton" name="pushButton">
                 <property name="geometry">
                  <rect>
                   <x>20</x>
                   <y>30</y>
                   <width>93</width>
                   <height>28</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Color_Space</string>
                 </property>
                </widget>
               </widget>
              </widget>
              <widget class="QWidget" name="widget_2" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QSplitter" name="splitter">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <widget class="QFrame" name="image_win">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>350</height>
                    </size>
                   </property>
                   <property name="cursor">
                    <cursorShape>CrossCursor</cursorShape>
                   </property>
                   <property name="toolTip">
                    <string>图片显示区域</string>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::Box</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <property name="lineWidth">
                    <number>2</number>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_13">
                    <item alignment="Qt::AlignHCenter">
                     <widget class="QLabel" name="show_label">
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                  <widget class="QWidget" name="widget" native="true">
                   <layout class="QVBoxLayout" name="verticalLayout">
                    <item>
                     <widget class="QTabWidget" name="tabWidget">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>100</height>
                       </size>
                      </property>
                      <property name="toolTip">
                       <string>输出区域</string>
                      </property>
                      <property name="tabPosition">
                       <enum>QTabWidget::North</enum>
                      </property>
                      <property name="tabShape">
                       <enum>QTabWidget::Rounded</enum>
                      </property>
                      <property name="currentIndex">
                       <number>0</number>
                      </property>
                      <property name="usesScrollButtons">
                       <bool>true</bool>
                      </property>
                      <property name="documentMode">
                       <bool>false</bool>
                      </property>
                      <property name="tabsClosable">
                       <bool>false</bool>
                      </property>
                      <property name="tabBarAutoHide">
                       <bool>true</bool>
                      </property>
                      <widget class="QWidget" name="dbug_tab">
                       <attribute name="title">
                        <string>运行输出</string>
                       </attribute>
                       <layout class="QVBoxLayout" name="verticalLayout_18">
                        <item>
                         <widget class="QTextEdit" name="debugEdit"/>
                        </item>
                       </layout>
                      </widget>
                      <widget class="QWidget" name="show_tab">
                       <attribute name="title">
                        <string/>
                       </attribute>
                       <layout class="QVBoxLayout" name="verticalLayout_19">
                        <item>
                         <widget class="QTextEdit" name="introductionEdit"/>
                        </item>
                       </layout>
                      </widget>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="win1">
       <layout class="QVBoxLayout" name="verticalLayout_9"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1102</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuyan">
    <property name="title">
     <string>图片</string>
    </property>
    <addaction name="lead_image"/>
    <addaction name="save_image"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="red_skin"/>
    <addaction name="blue_skin"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="statement"/>
    <addaction name="Algorithm_list"/>
   </widget>
   <addaction name="menuyan"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="lead_image">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../微信公众号图文/大四毕业设计/UI设计/皮肤主题素材/图标/add-bold.png</normaloff>../../微信公众号图文/大四毕业设计/UI设计/皮肤主题素材/图标/add-bold.png</iconset>
   </property>
   <property name="text">
    <string>打开图片</string>
   </property>
   <property name="toolTip">
    <string>插入图片</string>
   </property>
  </action>
  <action name="save_image">
   <property name="icon">
    <iconset>
     <normaloff>../../微信公众号图文/大四毕业设计/UI设计/皮肤主题素材/图标/download.png</normaloff>../../微信公众号图文/大四毕业设计/UI设计/皮肤主题素材/图标/download.png</iconset>
   </property>
   <property name="text">
    <string>导出并保存</string>
   </property>
   <property name="toolTip">
    <string>导出图片</string>
   </property>
  </action>
  <action name="red_skin">
   <property name="icon">
    <iconset>
     <normaloff>../../微信公众号图文/大四毕业设计/UI设计/皮肤主题素材/截图16.jpg</normaloff>../../微信公众号图文/大四毕业设计/UI设计/皮肤主题素材/截图16.jpg</iconset>
   </property>
   <property name="text">
    <string>红色皮肤</string>
   </property>
   <property name="toolTip">
    <string>海贼王皮肤</string>
   </property>
  </action>
  <action name="statement">
   <property name="icon">
    <iconset>
     <normaloff>../../微信公众号图文/大四毕业设计/UI设计/皮肤主题素材/图标/column-horizontal.png</normaloff>../../微信公众号图文/大四毕业设计/UI设计/皮肤主题素材/图标/column-horizontal.png</iconset>
   </property>
   <property name="text">
    <string>软件声明</string>
   </property>
   <property name="toolTip">
    <string>软件声明</string>
   </property>
  </action>
  <action name="blue_skin">
   <property name="icon">
    <iconset>
     <normaloff>../../微信公众号图文/大四毕业设计/UI设计/皮肤主题素材/截图15.jpg</normaloff>../../微信公众号图文/大四毕业设计/UI设计/皮肤主题素材/截图15.jpg</iconset>
   </property>
   <property name="text">
    <string>蓝色皮肤</string>
   </property>
   <property name="toolTip">
    <string>海贼王皮肤</string>
   </property>
  </action>
  <action name="Algorithm_list">
   <property name="icon">
    <iconset>
     <normaloff>../../微信公众号图文/大四毕业设计/UI设计/皮肤主题素材/图标/help.png</normaloff>../../微信公众号图文/大四毕业设计/UI设计/皮肤主题素材/图标/help.png</iconset>
   </property>
   <property name="text">
    <string>算法明细</string>
   </property>
   <property name="toolTip">
    <string>算法清单介绍</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
